src/v/cluster/tests/autocreate_non_replicable_test.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020 Vectorized, Inc.
```

Comment:
Nice attempt here, one big caveat though, since its been a while since you started this PR there have been some changes in dev that this will clash with and when merged these tests will fail. That is because a lot of the non replicable implementation details have been moved into coproc and thus now are now guarded with the `enable_coproc` feature flag.

So I think you should rebase against dev and set the `enable_coproc` flag to true on all node fixtures in the unit test cluster before the nodes start.

Reply:
This comment has not been resolved either.